Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.6.5-alpine3.7
+
+RUN apk update && apk upgrade && \
+    apk add --no-cache bash git openssh libffi-dev libffi pkgconf gcc g++ openssl openssl-dev shadow
+
+RUN useradd -u 9999 errbot
+
+Run  git clone https://github.com/cardoso/errbot-rocketchat
```

Comment:
Can you make this RUN command uppercase?

Reply:
Totally missed that 😊

Reply:
Woops I also totally missed that ( workdays are to long)